src/components/PlioQuestion.vue

```diff
@@ -73,11 +76,17 @@
           >
           ✓ सबमिट करें
           </button>
+          <submit-button-pointer 
+            v-if="!isTutorialComplete && !tutorialProgress['submit'] && !isDisabled && !isAnAnsweredQuestion">
+          </submit-button-pointer>
```

Comment:
The submit button pointer should appear only when 
(
the tutorial is not complete +
this particular step has not been seen before +
the submit button is not disabled +
this question has not been answered before
)

Reply:
Why the other conditions. Why not only the two tutorial conditions?